src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -40,13 +41,13 @@ class DebugHandlersListener implements EventSubscriberInterface
     private $hasTerminatedWithException;
 
     /**
-     * @param callable|null        $exceptionHandler A handler that will be called on Exception
-     * @param LoggerInterface|null $logger           A PSR-3 logger
-     * @param array|int            $levels           An array map of E_* to LogLevel::* or an integer bit field of E_* constants
-     * @param int|null             $throwAt          Thrown errors in a bit field of E_* constants, or null to keep the current value
-     * @param bool                 $scream           Enables/disables screaming mode, where even silenced errors are logged
-     * @param string|array         $fileLinkFormat   The format for links to source files
-     * @param bool                 $scope            Enables/disables scoping mode
+     * @param callable|null                 $exceptionHandler A handler that will be called on Exception
+     * @param LoggerInterface|null          $logger           A PSR-3 logger
+     * @param array|int                     $levels           An array map of E_* to LogLevel::* or an integer bit field of E_* constants
+     * @param int|null                      $throwAt          Thrown errors in a bit field of E_* constants, or null to keep the current value
+     * @param bool                          $scream           Enables/disables screaming mode, where even silenced errors are logged
+     * @param string|FileLinkFormatter|null $fileLinkFormat   The format for links to source files
```

Comment:
forwarded to the `ExceptionHandler` which supports only `string|FileLinkFormatter`
https://github.com/symfony/symfony/blob/2e4020c4303b395c1997d4a472602e41a9216a03/src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php#L140